angular.json

```diff
@@ -1227,7 +1227,7 @@
                 {
                   "type": "initial",
                   "maximumWarning": "2mb",
-                  "maximumError": "5mb"
```

Comment:
Do you know what change caused the budget to jump like this? Something must be misconfiured

Reply:
Caused by plotly. Makes sense

Reply:
Not sure. Its much over this.